CONTRIBUTING.adoc

```diff
@@ -26,3 +27,36 @@ Settings`, it will set this policy across all of your IDEA projects.
 * IDEA's "Optimize imports on the fly" option turned on interferes with the Eclipse code formatter
 import optimization. Consider disabling the option if optimization does not yield the expected
 results.
+
+= Contribution workflow
```

Comment:
I think we need to make it clear that this workflow applies to project committers.

Community contributions should follow a workflow like this:

# Contributing A Patch

* Submit an issue describing your proposed change to the repo in question.
* The repo owners will respond to your issue promptly.
* If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement [link].
* Fork the repo, develop and test your code changes.
* Ensure that your code adheres to the existing style in the sample to which you are contributing. Refer to the Spring Style Guide [link] for the recommended coding standards for this organization.
* Ensure that your code has an appropriate set of unit tests which all pass.
* Submit a pull request.

Reply:
+1. We don't need to tell target contributor how to press "squash & merge".

Reply:
If the workflow in this PR only applies to project committers, I wonder if it's worth to have it in here at all? After all, projects committers should have access to the Google Doc the workflow was based in.